presentation-api/receiving-ua/common.js

```diff
@@ -7,4 +7,40 @@
     toController: '0c382524-5738-4df0-837d-4f53ea8addc2',
     toReceiver: 'a9618cd1-ca2b-4155-b7f6-630dce953c44'
   }
+
+  // handle a test result received from a receiving page
```

Comment:
Okay. I have fixed them.

* If any `match` fails, `assert_unreached` is called with a default message.
* `JSON.parse` is now wrapped in `try... catch`.